fossid-webapp-client/src/main/kotlin/api/identification/markedAsIdentified/MarkedAsIdentifiedFile.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.fossid.api.identification.markedAsIdentified
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonUnwrapped
+
+import org.ossreviewtoolkit.fossid.api.identification.common.Component
+import org.ossreviewtoolkit.fossid.api.summary.License
+import org.ossreviewtoolkit.fossid.api.summary.Summarizable
+import org.ossreviewtoolkit.fossid.api.summary.SummaryIdentifiedFile
+
+class MarkedAsIdentifiedFile : Summarizable {
+    var comment: String? = null
+
+    @JsonProperty("identification_id")
+    var identificationId: Int? = null
+
+    @JsonProperty("identification_copyright")
+    var identificationCopyright: String? = null
+
+    @JsonProperty("is_distributed")
+    var isDistributed: Int? = null
+
+    @JsonProperty("row_id")
+    var rowId: Int? = null
+
+    @JsonProperty("user_name")
+    var userName: String? = null
+
+    @JsonProperty("user_surname")
+    var userSurname: String? = null
+
+    @JsonProperty("user_username")
+    var userUsername: String? = null
+
+    @JsonUnwrapped(prefix = "component_")
+    var component: Component? = null // TODO how to get the additional components ? Only one is returned
+
+    @JsonUnwrapped(prefix = "file_")
+    var file: File? = null
+
+    override fun toSummary(): SummaryIdentifiedFile {
+        val licenses = kotlin.run {
+            if (file!!.licenses == null) {
+                emptyList()
+            } else {
+                file!!.licenses!!.values.map {
+                    License(
+                            identifier = it.file!!.licenseIdentifier!!,
+                            name = it.file!!.licenseName,
+                            origin = it.type!!
+                    )
+                }
+            }
+        }
+        return SummaryIdentifiedFile(
+                path = file!!.path!!,
+                comment = comment,
+                licences = licenses)
```

Comment:
Looks like code duplication compared to `fossid-webapp-client/src/main/kotlin/api/identification/identifiedFiles/IdentifiedFile.kt`. Please look into ways to avoid this duplication.

Reply:
As I wrote in another response, since I don't have access to FossID data model, I don't feel comfortable consolidating now the classes of the model.
So currently I went with the solution of isolating the data model of each call to FossID server, at the price of some duplication.

When data model is released by FossID team, I will be happy to update the model in ORT.